homeassistant/components/deako/config_flow.py

```diff
@@ -0,0 +1,26 @@
+"""Config flow for deako."""
+
+from pydeako.discover import DeakoDiscoverer, DevicesNotFoundException
+
+from homeassistant.components import zeroconf
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_entry_flow
+
+from .const import DOMAIN, NAME
+
+
+async def _async_has_devices(hass: HomeAssistant) -> bool:
+    """Return if there are devices that can be discovered."""
+    _zc = await zeroconf.async_get_instance(hass)
```

Comment:
If we're using zeroconf, why aren't we setting up the devices with 1 device = 1 config entry? (note, I don't have much experience with zeroconf, but please explain a bit more on how the device works)

Reply:
Yeah for sure. Good question. We have some devices that can discovered with mdns. We pick one to the be "bridge". It can be any of them that are on wifi at the time. That device is then responsible for bridging tcp to ble as not all deako devices are on wifi at all times. Once a device is selected to be the bridge, it will stay on wifi as the local integration device. All other devices periodically get on wifi to check in, send logs, etc. 

Since we only really connect to one device, one config entry. All other devices are then exposed through that device using the local API.

Reply:
What if that bridge node breaks down? What will happen?